pkgs/development/tools/database/sqlitebrowser/default.nix

```diff
@@ -1,30 +1,35 @@
-{ lib, stdenv, mkDerivation, fetchFromGitHub, cmake
-, qtbase, qttools, sqlcipher, wrapGAppsHook3, qtmacextras
+{ lib, stdenv, fetchFromGitHub, cmake
+, qtbase, qttools, sqlcipher, wrapQtAppsHook, qtmacextras
 }:
 
-mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "sqlitebrowser";
   version = "3.13.0";
 
   src = fetchFromGitHub {
-    owner = pname;
-    repo = pname;
-    rev = "v${version}";
+    owner = "sqlitebrowser";
+    repo = "sqlitebrowser";
+    rev = "v${finalAttrs.version}";
     sha256 = "sha256-2U0jnL2hmrxynMxEiObl10bKFAFlCrY2hulZ/Ggqimw=";
   };
 
+  patches = lib.optional stdenv.isDarwin ./macos.patch;
```

Comment:
`.app` bundles are typically installed on macOS by copying them to `/Applications/...`, but I wouldn't want nix do that for me...

If this was a home-manager installation, it would be a different thing, but the .app bundle is broken anyway without all the qt-related wrapping.

This PR makes the command line package "just work" as it does on linux.